src/vs/workbench/contrib/interactiveEditor/browser/interactiveEditorController.ts

```diff
@@ -412,6 +413,9 @@ export class InteractiveEditorController implements IEditorContribution {
 		});
 
 		const typeListener = this._zone.widget.onDidChangeInput(() => {
+			if (this._zone.widget.value.includes(interactiveEditorHelpIntro)) {
```

Comment:
👃 this isn't a nice solution to the problem. IMO it would be better if we have some state that tells us that we in this mode, that would also prevent https://github.com/microsoft/vscode-internalbacklog/issues/4175